lib/screens/homepage.dart

```diff
@@ -17,8 +17,21 @@ class _HomePageState extends State<HomePage> {
       // TODO: @Luis to create the body layout
       body: Container(),
 
-      // TODO: @Brano to create the bottom navigation bar
-      bottomNavigationBar: BottomNavigationBar(items: const []),
+      bottomNavigationBar:
+          BottomNavigationBar(items: const <BottomNavigationBarItem>[
```

Comment:
Hi Brano,

I think it would be better if we abstract this component as a class inside the ./components folder. You can just import it inside this file. What do you think?

Reply:
Hey Luis!

I refactored that part, so pls take a look 🙂